spec/rubocop/cop/rspec/pending_without_reason_spec.rb

```diff
@@ -65,11 +97,39 @@
     end
   end
 
+  context 'when pending by pending step without reason ' \
+          'and not inside example with block' do
+    it 'registers no offense' do
+      expect_no_offenses(<<~RUBY)
+        FactoryBot.define do
+          factory :task do
+            pending { true }
+          end
+        end
+      RUBY
+    end
+  end
+
+  context 'when skipped by skip step without reason ' \
+          'and not inside example with block' do
+    it 'registers no offense' do
+      expect_no_offenses(<<~RUBY)
+        FactoryBot.define do
+          factory :task do
+            skip { true }
+          end
+        end
+      RUBY
+    end
+  end
+
   context 'when pending with receiver' do
     it 'registers no offense' do
       expect_no_offenses(<<~RUBY)
-        it 'does something' do
-          Foo.pending
+        RSpec.describe Foo do
+          it 'does something' do
+            Foo.pending
```

Comment:
```suggestion
            Foo.skip
```
I suggest grouping those "no offence" examples unless there's a reason to have them separate (like a block with a single statement instead of a `begin` node for multi-statement).

Reply:
Indeed it is. I inherited the way the original spec was done, but I will try to put it together.